web/src/pages/project/ValueFilter.tsx

```diff
@@ -0,0 +1,252 @@
+        setDefaultFilterType(newFilterType)
+        const newOperator = getOperatorFromFilterType(newFilterType)
+        if (!newOperator) return console.warn('No operator from query type', newFilterType)
+        if (tempValue) postValue(newOperator, tempValue)
+    }
+
+    const postValue = (_operator: string, value: string | undefined) => {
+        const f: GenericFilterAny | undefined = !!value ? { [_operator]: value } : undefined
+
+        // deep copy
+        const newFilter = JSON.parse(JSON.stringify(props.filterValues))
+
+        const base = newFilter[category] || {}
+        if (isMeta) {
+            const oldMeta = base.meta || {}
+            newFilter[category] = { ...base, meta: { ...oldMeta, [name]: f } }
+        } else {
+            newFilter[category] = { ...base, [name]: f }
+        }
+
+        updateFilterValues(newFilter)
+    }
+
+    const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault()
+        postValue(operator!, tempValue)
+    }
+
+    let input: React.ReactElement = <p>{message}</p>
+
+    if (!disabled) {
+        input = (
+            <Input
+                icon="search"
+                placeholder={`Filter ${name}...`}
+                name={name}
+                value={tempValue}
+                onChange={(e) => setTempValue(e.target.value)}
+                size={props.size}
+                labelPosition="left"
+                onSubmit={onSubmit}
+            >
+                {/* label={ */}
+                <Label>
+                    <Dropdown
+                        value={queryType || ''}
+                        onChange={(_, { value }) =>
+                            updateQueryType(value as ProjectParticipantGridFilterType)
+                        }
+                        options={options.map((q) => ({
+                            key: q,
+                            text: getDisplayNameFromFilterType(q),
+                            value: q,
+                        }))}
+                    />
+                </Label>
+                <input />
+                {isHighlighted && (
+                    <Label style={{}}>
+                        <IconButton
+                            style={{ padding: '0', height: '12px', width: '12px' }}
+                            onClick={() => {
+                                postValue(operator, undefined)
+                                setTempValue('')
+                            }}
+                        >
+                            <CloseIcon style={{ padding: '0', margin: '0' }} />
+                        </IconButton>
+                    </Label>
+                )}
+
+                {/* } */}
+            </Input>
+        )
+    }
+    // debugger
+
+    return (
+        <Form onSubmit={onSubmit}>
+            <Form.Group inline style={{ padding: 0, margin: 0 }}>
+                <Form.Field style={{ padding: 0, margin: 0 }}></Form.Field>
```

Comment:
Is this empty field required here?

Reply:
Nope, just bad copy paste from the original code